proto/regen/ecocredit/basket/v1/state.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+import "cosmos/orm/v1alpha1/orm.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
+
+// Basket represents a basket in state.
+message Basket {
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 1,
+        primary_key: {fields: "id", auto_increment: true}
+        index: {id: 1, fields: "basket_denom", unique: true}
+    };
+
+    // id is the uint64 ID of the basket. It is used internally for reducing
+    // storage space.
+    uint64 id = 1;
+
+    // basket_denom is the basket denom.
+    string basket_denom = 2;
+
+    // disable_auto_retire indicates whether or not the credits will be retired upon withdraw from the basket.
+    bool disable_auto_retire = 3;
+
+    // credit_type_name is the name of the credit type this basket is able to hold.
+    string credit_type_name = 4;
+
+    // min_start_date is the earliest start date for batches of credits allowed
+    // into the basket.
+    google.protobuf.Timestamp min_start_date = 5;
```

Comment:
```suggestion
    google.protobuf.Timestamp start_date = 5;
```

Reply:
Let's remove `min`, I think `start_date` is enough.

Reply:
I would prefer to keep `min` but no strong preference.

Reply:
let's keep `min_`. it's not really start date but the minimum or earliest start date. could be `earliest_start_date` but that's even longer